.github/workflows/codeql.yml

```diff
@@ -0,0 +1,82 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "master", "18.06", "18.09", "19.03", "[a-zA-Z0-9][a-zA-Z0-9].[a-zA-Z0-9]", "[a-zA-Z0-9][a-zA-Z0-9].[a-zA-Z0-9][a-zA-Z0-9]" ]
```

Comment:
We can remove the 19.03 and older branches; they're no longer maintained (I may have locked them as well)

(FWIW I'm considering to use different naming schemes for release branches; `release/XXX` for actively maintained release branches, and `archive/XXX` for branches that are no longer maintained - assuming those branches contain possibly code that was not in the latest tag from the branch)

Reply:
> Canned replies FTW 😂

Lol yes

> We can remove the 19.03 and older branches; they're no longer maintained (I may have locked them as well)

Sure, let's leave it just on `master`! We can update it if you decide to adopt the new release branch naming convention.